dist/index.js

```diff
@@ -0,0 +1,7015 @@
+/******/ (() => { // webpackBootstrap
```

Comment:
What process generates this file? do we need to run webpack locally and push it?

Reply:
hasky, yes we do, although it does not have to be "locally". I'd say it shall be automated using github actions which can be included to a separate PR.

Reply:
Let's create an issue for that?

Reply:
Done, see #3